fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -2296,19 +2296,49 @@ public ParseNode visitAdminSetConfig(StarRocksParser.AdminSetConfigContext conte
     }
 
     @Override
-    public ParseNode visitSetVar(StarRocksParser.SetVarContext ctx) {
-        Expr expr = (Expr) visit(ctx.expression());
-        String variable = ctx.IDENTIFIER().getText();
-        if (ctx.varType() != null) {
-            return new SetVar(getVariableType(ctx.varType()), variable, expr);
+    public ParseNode visitSetStatement(StarRocksParser.SetStatementContext context) {
+        List<SetVar> propertyList = visit(context.setVar(), SetVar.class);
+        return new SetStmt(propertyList);
+    }
+
+    @Override
+    public ParseNode visitSetVar(StarRocksParser.SetVarContext context) {
+        if (context.AT().isEmpty()) {
+            Expr expr = (Expr) visit(context.setExprOrDefault());
+            String variable = ((Identifier) visit(context.identifier())).getValue();
+            if (context.varType() != null) {
+                return new SetVar(getVariableType(context.varType()), variable, expr);
+            } else {
+                return new SetVar(variable, expr);
+            }
+        } else if (context.AT().size() == 1) {
+            Expr expr = (Expr) visit(context.expression());
```

Comment:
I'm not familiar with it, could you explain why one `@` calling `context.expression()` while zero or two `@` calling `context.setExprOrDefault()`